app/Models/Traits/Es/TeamSearch.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+declare(strict_types=1);
+
+namespace App\Models\Traits\Es;
+
+use App\Libraries\BBCodeForDB;
+use App\Libraries\BBCodeFromDB;
+
+trait TeamSearch
+{
+    use BaseDbIndexable;
+
+    public static function esIndexingQuery()
+    {
+        return static::query();
+    }
+
+    public function toEsJson()
+    {
+        $document = [];
+        foreach (static::esMappings() as $field => $mapping) {
+            $document[$field] = match ($field) {
+                'description' => BBCodeFromDB::removeBBCodeTags(new BBCodeForDB($this->description ?? '')->generate()),
```

Comment:
need to decode html entities as well, or apply the `html_strip` char_filter to the field. Also maybe `removeBlockQuotes` as well unless we still want the content in the quotes to be searchable, then maybe stripping the html tags that get added

Reply:
the quotes stay because there's no parent